src/reducers/entities/threads.test.js

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {ThreadTypes} from 'action_types';
+import deepFreeze from 'utils/deep_freeze';
+import threadsReducer from './threads';
+
+describe('threads', () => {
+    describe('RECEIVED_THREADS', () => {
```

Comment:
Missing a the tests here around the other events that can happen on the threads reducers, like the `FOLLOW_CHANGED_THREAD` and the `LEAVE_TEAM`.

And missing tests for the other existing reducers like `threadsReducer`,  `threadsInTeamReducer` and `countsReducer`.